src/modchannels.cpp

```diff
@@ -0,0 +1,102 @@
+/*
+Minetest
+Copyright (C) 2017 nerzhul, Loic Blot <loic.blot@unix-experience.fr>
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation; either version 2.1 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License along
+with this program; if not, write to the Free Software Foundation, Inc.,
+51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+#include "modchannels.h"
+#include <algorithm>
+
+bool ModChannel::register_consumer(u16 peer_id)
```

Comment:
We have other standards for function names. Mind changing them?

Reply:
you mean ModChannel standard against Client/Server standard ?

Reply:
`register_consumer` -> `registerConsumer` that's what I mean.

Reply:
okay for the camelCase